docs/configure/triggers.md

```diff
@@ -204,8 +255,113 @@ The following attributes are available for triggers:
+| `notifications` |  object | **Required** | The notifications object: <ul><li>`type`: The type of the notification. Options: `webhook`, `email`</li><li>`value`: The URL to send the request to or the email address to notify.</li><li>`seconds_between_notifications`: The interval between notifications in seconds.</li></ul> |
+
+#### Conditions
+
+The `conditional` object for the `conditional_data_ingested` trigger includes the following options:
+
+| Name                  | Type   | Required?    | Description                                                                                                                                                                                                                                                                                                                                                                       |
+| --------------------- | ------ | ------------ | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `data_capture_method` | string | **Required** | The method of data capture to trigger on. <br> Example: `sensor:<name-of-component>:Readings`.                                                                                                                                                                                                                                                                                    |
+| `condition`           | object | Optional     | Any additional conditions for the method to fire the trigger. Leave out this object for the trigger to fire any time there is data synced. <br> Options: <ul><li>`evals`:<ul><li>`operator`: Logical operator for the condition. </li><li>`value`: Can be a number, string or another object like a key-value pair that specifies the shape of the readings. </li></ul></li></ul> |
+
+Options for `operator`:
+
+| Name  | Description              |
+| ----- | ------------------------ |
+| `lt`  | Less than                |
+| `gt`  | Greater than             |
+| `lte` | Less than or equal to    |
+| `gte` | Greater than or equal to |
+| `eq`  | Equals                   |
+| `neq` | Does not equal           |
+
+Examples:
+
+{{< tabs >}}
+{{% tab name="No nesting" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+"condition": {
+  "evals": [
+    {
+      "operator": "lt",
+      "value": 20
+    }
+  ]
+}
+```
+
+This eval would trigger for the following sensor reading:
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "readings": {
+    "key": 10
```

Comment:
no example so not sure if this is accurate (not sure whether there would be a key or not)-- can anyone confirm?

Reply:
all the examples I have been able to configure in the ui do have keys

Reply:
@RoxyFarhad

Reply:
yeah I think they all have the reading keys right now